runconfig/opts/parse.go

```diff
@@ -521,6 +529,20 @@ func parseSecurityOpts(securityOpts []string) ([]string, error) {
 	return securityOpts, nil
 }
 
+// parse storage options per container into a map
+func parseStorageOpts(storageOpts []string) (map[string]string, error) {
+	m := make(map[string]string)
+	for _, option := range storageOpts {
+		if strings.Contains(option, "=") {
```

Comment:
@thaJeztah just because I want to be sure, opts from now on should only contains equal sign (for *new* opts) otherwise for the _old_ opts we accept both equal and column separator right?